vms/platformvm/state/state.go

```diff
@@ -1457,14 +1457,13 @@ func (s *state) loadCurrentValidators() error {
 		}
 
 		metadataBytes := validatorIt.Value()
-		defaultStartTime := stakerTx.StartTime()
 		metadata := &validatorMetadata{
 			txID: txID,
 			// use the start values as the fallback
 			// in case they are not stored in the database
 			// Note: we don't provide [LastUpdated] here because we expect it to
 			// always be present on disk.
-			StakerStartTime: defaultStartTime.Unix(),
+			StakerStartTime: stakerTx.StartTime().Unix(),
```

Comment:
Post Durango txs will persist their start time, hence they won't use this fallback value. PreDurango, txs may have their StartTime not stored, so they will use it